pinot-spi/src/main/java/org/apache/pinot/spi/stream/StreamMessageDecoder.java

```diff
@@ -33,14 +34,17 @@
 @InterfaceStability.Stable
 public interface StreamMessageDecoder<T> {
 
-  static final String RECORD_EXTRACTOR_CONFIG_KEY = "recordExtractorClass";
+  String RECORD_EXTRACTOR_CONFIG_KEY = "recordExtractorClass";
```

Comment:
why remove the static final?

Reply:
variables in an interface are implicitly static final. This used to be an abstract class before my previous refactoring, and the `static final` got carried forward